block/headerfieldsetter.go

```diff
@@ -3,8 +3,8 @@ package block
 import (
 	"math/big"
 
-	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/harmony-one/harmony/common"
+	types "github.com/harmony-one/harmony/ethcore/types"
```

Comment:
why ethcore/types? can it be merged into our own harmony/core/types?

Reply:
harmony/block/headerfieldsetter.go will import github.com/ethereum/go-ethereum/core/types.
harmony/core/type/block.go will import github.com/harmony-one/harmony/block.
so if we merge /go-ethereum/core/types to our own harmony/core/types, it will cause loop import.
harmony/core/type --> harmony/block --> harmony/core/type